tests/Command/ArchiveTokenCommandTest.php

```diff
@@ -33,10 +30,10 @@ class ArchiveTokenCommandTest extends KernelTestCase
 
     protected function setUp(): void
     {
-        $this->archivist = $this->prophesize(ArchivistInterface::class);
+        $this->archivist = $this->createMock(ArchivistInterface::class);
```

Comment:
🙏🏻

Reply:
Yeah... I wasn't about to do it, but I had very annoying exceptions with prophecy in the test suite, so I removed

```
Prophecy\Exception\Doubler\ClassMirrorException: Doubling intersection types is not supported
```